keylime/tenant.py

```diff
@@ -953,7 +959,7 @@ def do_quote(self):
                 response_body = response.json()
 
             except Exception as e:
-                if response.status_code in (503, 504):
+                if response is not None and response.status_code in (503, 504):
```

Comment:
```suggestion
                if response is None or response.status_code in (503, 504):
```
If we don't get anything back than also retry

Reply:
thanks!